100.en.adoc

```diff
@@ -2,116 +2,118 @@
+The <<figure-01,flowchart>> outlines the processing steps involved in publishing amplicon-derived molecular biodiversity data in repositories such as GBIF and national biodiversity data platforms, including those built on the ALA platform. This guide’s focus is primarily on the steps following the arrival of raw <<fastq,FASTQ>> sequences from the sequencing step. By familiarizing themselves with the flowchart—and noting any steps that appear familiar or unclear—users will be able to navigate the content included in the guide.
 
 [[figure-01]]
 .Overall workflow for sequence-derived biodiversity data as described in this guide.
 image::img/web/figure-01.png[]
 
-We have done our best to present the information in this guide so that it is useful for anyone included within the audiences described above, but background reading (e.g. https://www.gbif.org/publishing-data[GBIF quick guide to data publishing]) may be required in certain cases.
+We have done our best to present the information in this guide so that it is useful for each of the audiences described above, but background reading (e.g. https://www.gbif.org/publishing-data[GBIF quick guide to data publishing]) may be required in certain cases.
 
 === Introduction to sequence-derived occurrence data
 
-Sequence derived biological occurrence data includes environmental DNA (eDNA) (DNA extracted from environmental samples or, rather than from individual organisms (https://doi.org/10.1016/j.biocon.2014.11.019[Thomsen and Willerslev 2015^]) and, from bulk samples comprising many individuals (e.g. plankton samples or Malaise trap samples consisting of multiple individuals from many species). The most common form of sequence-derived occurrence data is currently that derived from eDNA, and because the analysis methods and end products are very similar between the different sample sources, the discussion below will focus on eDNA (<<category-i-sequence-derived-occurrences,§2.1.1>> and <<category-ii-enriched-occurrences,§2.2.1>>), noting that it is applicable to the other sources.
+Sequence derived biological occurrence data includes information derived from sequencing individual organisms, but also from environmental DNA (eDNA, i.e. DNA extracted from environmental samples, https://doi.org/10.1016/j.biocon.2014.11.019[Thomsen and Willerslev 2015^]) and from bulk samples comprising many individuals (e.g. plankton samples or Malaise trap samples consisting of multiple individuals from many species). Currently, the greatest volume of sequence-derived occurrence data derives from eDNA. Since analytical methods and end products are laregly similar for all sample sources, the discussion below will focus on eDNA (<<category-i-sequence-derived-occurrences,§2.1.1>> and <<category-ii-enriched-occurrences,§2.2.1>>), noting that the outline is applicable to the other sources.
```

Comment:
- add hyphen to 'Sequence-derived'
- remove unnecessary section numbers (now added as shortref style through the whole document